### DISCUSSION TITLE
ArrayCards.Object如何实现第一项不能删除

### DISCUSSION THREAD

**Discussion by Aerojin** [alibaba/formily#2428] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/3827750/141297676-c216d373-2ed3-47db-ae94-94afc24d7d1e.png)

**Comment by janryWang** [comment, marked as answer]
https://codesandbox.io/s/trusting-dirac-w990w?file=/App.tsx

**Reply by baojianlsy** [reply]
<img width="1164" alt="image" src="https://user-images.githubusercontent.com/96938140/216285243-0c58a689-80d3-49f1-8239-f8674ecdc3bc.png">
我这样引入会报错呢，求助
